R/az_daily.R

```diff
@@ -89,9 +89,23 @@ az_daily <- function(station_id = NULL, start_date = NULL, end_date = NULL) {
     #convert NAs
     dplyr::mutate(
       dplyr::across(
-        tidyselect::where(is.numeric),
+        c(tidyselect::where(is.numeric)),
         function(x)
-          dplyr::if_else(x %in% c(-999, -9999, -99999, -7999, 999, 999.9, 9999), NA_real_, x))
+          dplyr::if_else(x %in% c(-999, -9999, -99999, -7999, 999, 999.9, 9999), NA_real_, x)
+      )
+    ) %>%
+    dplyr::mutate(
+      dplyr::across(
+        wind_2min_timestamp,
+        function(x)
+          dplyr::if_else(x == as.character(-99999), NA_character_, x)
+      )
+    ) %>%
+    dplyr::mutate(
+      wind_2min_timestamp = lubridate::with_tz(
+        lubridate::parse_date_time(.data$wind_2min_timestamp, orders = "ymdHMSz"),
```

Comment:
ahhh, I understand this now.  So pars_date_time does actually understand the timezone, it just converts it automatically to UTC and `with_tz()` converts it _back_ to MST.  Cool!